cypress/utils/commands.js

```diff
@@ -633,6 +643,60 @@ Cypress.Commands.add(
   }
 );
 
+Cypress.Commands.add(
+  'selectFromDataSourceSelectable',
+  (dataSourceTitle, dataSourceId) => {
+    cy.get('#dataSourceSelectableContextMenuPopover').click();
+    cy.checkDefaultDataSourceExist();
+    cy.getElementByTestId('dataSourceSelectable')
+      .find('input')
+      .clear('{backspace}')
+      .type(dataSourceTitle);
+    cy.wait(1000);
+    let dataSourceElement;
+    if (dataSourceId) {
+      dataSourceElement = cy.get(`#${dataSourceId}`);
+    } else if (dataSourceTitle) {
+      dataSourceElement = cy
+        .get('.euiSelectableListItem')
+        .contains(dataSourceTitle)
+        .closest('.euiSelectableListItem');
+    }
+    dataSourceElement.click();
+    // Close data source selectable manually if no data source element need to be clicked
+    if (!dataSourceElement) {
+      cy.getElementByTestId('dataSourceSelectable').last('button').click();
+    }
+  }
+);
+
+Cypress.Commands.add('checkDefaultDataSourceExist', () => {
+  cy.contains('li.euiSelectableListItem', 'default-ds')
+    .should('exist') // Ensure the list item exists
+    .within(() => {
+      // Verify the 'Default' badge exists within the same list item
+      cy.get('span.euiBadge__text').should('exist').and('contain', 'Default'); // Ensure the badge contains the text 'Default'
+    });
+});
+
+Cypress.Commands.add('viewDataSourceAggregatedView', () => {
+  cy.get('#dataSourceSViewContextMenuPopover').click();
+  cy.wait(10000);
+
+  cy.get('.dataSourceAggregatedViewOuiPanel').within(() => {
+    // Check if the Local cluster is selected
+
+    cy.contains('RemoteDataSourceNoAuth').should('be.visible');
```

Comment:
shall we make `RemoteDataSourceNoAuth ` a constant? I've been seeing this text being used in many places in codebase.